Description: Fix postinst warning
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/1060444
Forwarded: no
Last-Update: 2024-05-06

--- networkx-3.2.1.orig/networkx/readwrite/tests/test_gml.py
+++ networkx-3.2.1/networkx/readwrite/tests/test_gml.py
@@ -554,8 +554,8 @@ graph
             "directed 1 multigraph 1 ]"
         )
         assert_parse_error(
-            "graph [edge [ source u'u\4200' target u'u\4200' ] "
-            + "node [ id u'u\4200' label b ] ]"
+            r"graph [edge [ source u'u\4200' target u'u\4200' ] "
+            + r"node [ id u'u\4200' label b ] ]"
         )
 
         def assert_generate_error(*args, **kwargs):
